packages/main/src/plugin/index.ts

```diff
@@ -1300,6 +1303,9 @@ export class PluginSystem {
         }
         try {
           await providerRegistry.createKubernetesProviderConnection(internalProviderId, params, logger, token);
+        } catch (error) {
+          logger.error(error);
```

Comment:
I wonder why you need this. Isn't it already defined in the create function? It rejects in two cases, when the action is cancelled or if the process ends with an `exitcode` different to 0 and in both cases the `logger.error` is invoked. Doesn't it work?

Reply:
ah if it fails before invoking the runCliCommand... but then you will see a double error if you e.g cancel the action, no? 🤔

Reply:
you can reproduce the issue by throwing an error in the createIngressController method of the kind extension

Reply:
ok for me, got it 👍

Reply:
If I cancel I have:
![image](https://user-images.githubusercontent.com/436777/231711772-cd4da6c1-b086-4989-a83a-57fd53f04329.png)